static/assets/js/entity-reference/app.1.1.js

```diff
@@ -22,11 +22,13 @@
         this.apiTitle = '';
         this.showErrorMessage = false;
 
-        bbWait.beginPageWait({});
+        this.$onInit = function () {
```

Comment:
nitpick: can you remove the space before `()` to be consistent w/ the other `function() {}` declarations?